### ISSUE TITLE
xrncli q account <addr> textual output doesn't show balance

### ISSUE THREAD

**Issue by joe-bowman** [regen-network/regen-ledger#68]
**Describe the bug**
When querying account balances with `xrncli q account <addr>` the default (text) mode fails to show the balance in the account:

**To Reproduce**
Using version 0.4.1:
1. xrncli q account xrn:140y8m6r7s40mvmz6g5dqrsrfvfkq5m8cu30tnv

Returns:
```
$ xrncli q account xrn:140y8m6r7s40mvmz6g5dqrsrfvfkq5m8cu30tnv
address: xrn:140y8m6r7s40mvmz6g5dqrsrfvfkq5m8cu30tnv
coins:
- denom: seed
  amount: {}
pubkey:
- 2
- 76
- 145
- 39
- 14
- 136
- 244
- 214
- 118
- 38
- 18
- 226
- 137
- 252
- 241
- 161
- 43
- 7
- 38
- 94
- 202
- 129
- 143
- 35
- 30
- 110
- 11
- 248
- 195
- 168
- 55
- 206
- 215
accountnumber: 1078
sequence: 14
```

Note:

```
coins:
- denom: seed
  amount: {}
```

**Expected behavior**
The same command with `--output=json` shows the true balance:
```
$ xrncli q account xrn:140y8m6r7s40mvmz6g5dqrsrfvfkq5m8cu30tnv --output=json
{"type":"auth/Account","value":{"address":"xrn:140y8m6r7s40mvmz6g5dqrsrfvfkq5m8cu30tnv","coins":[{"denom":"seed","amount":"1000000822"}],"public_key":{"type":"tendermint/PubKeySecp256k1","value":"AkyRJw6I9NZ2JhLiifzxoSsHJl7KgY8jHm4L+MOoN87X"},"account_number":"1078","sequence":"14"}}
```

Note:
```
"coins": [{"denom":"seed","amount":"1000000822"}]
```

**Comment by aaronc** [comment]
Thanks @joe-bowman. This is an upstream issue with the version of the Cosmos SDK that we used for this testnet. It will be fixed when we upgrade the underlying Cosmos SDK version which I think is only safe to do after this testnet. (Unless we upgrade the Cosmos SDK in our upgrade test, but we decided not to go through the complexity of a 0.36 upgrade).

**Comment by aleem1413** [comment]
This issue addressed in #112
